neurogym/envs/annubes.py

```diff
@@ -19,6 +18,7 @@ def __init__(
         n_outputs=2,
         output_behavior=None,
         noise_std=0.01,
+        reward_dict=None,
```

Comment:
The template env just calls it `rewards`. I think it would be good to stick with the same nomenclature.
This will also need to be adjusted below.

```suggestion
        rewards=None,
```